draft-schinazi-connect-udp-listen.md

```diff
@@ -145,12 +142,12 @@ octets" in {{UDP}}).
 ## The connect-udp-listen Header Field {#hdr}
 
 The "connect-udp-listen" header field is an Item Structured Field, see {{Section
-3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST be an Integer; any other value
-type MUST be handled as if the field were not present by recipients (for
-example, if this field is included multiple times, its type will become a List
-and the field will therefore be ignored). This document does not define any
+3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST be an Integer and set as the Context ID. Any other value
```

Comment:
This sentence was following the usual structured header boilerplate, I'd rather add the fact that the value is the context ID as a separate sentence.